src/invidious/views/watch.ecr

```diff
@@ -26,7 +26,7 @@
 <meta name="twitter:player" content="<%= HOST_URL %>/embed/<%= video.id %>">
 <meta name="twitter:player:width" content="1280">
 <meta name="twitter:player:height" content="720">
-<link rel="alternate" href="https://www.youtube.com/watch?v=<%= video.id %>">
+<link rel="alternate noreferrer noopener" href="https://www.youtube.com/watch?v=<%= video.id %>">
```

Comment:
Please revert this change: `noreferrer` and `noopener` are not allowed on `<link>` elements:
https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel